### ISSUE TITLE
No Solana to Ethereum

### ISSUE THREAD

**Issue by RoRo-1** [solana-labs/oyster#346]
Hello!
Basically the bridge allows the switch from Ethereum to Solana but not the other way around (Solana --> Ethereum) and the token is stuck in the wallet.
My transaction.
Signature: 4HASfneNesXHg5qJ2CZNU3aknBnhdvsqXqSJMpX6dLRg6WU2PFFrxiV85fJMegychUHWYiB5v84oopecNB61irf1 
How can I swap or send into the Ethereum Network my wWBTC tokens?
Thanks

**Comment by RoRo-1** [comment]
[No-transfert-fromwormholeV1](https://user-images.githubusercontent.com/7474708/133911118-f0532d9c-1723-4d45-9603-8a0743c89dde.jpg)

)

**Comment by RoRo-1** [comment]
V2 does not allow me to transfer wWBTC because tell me my asset is "not eligible for transfer."
V1 does not allow me to transfer from Solana to Ethereum (which is what I want). You can see a continuous loading even for more than an hour without success.
The wWBT token is currently locked in my Phantom Wallet.
Can you guys help me please?
Thank you

**Comment by yamijuan** [comment]
Looks like the UI is rounding Up your balance. Your current balance is `0.06446857` but you are trying to transfer `0.06446900`. Make sure to use the correct balance and try again.

**Comment by RoRo-1** [comment]
Trying it on V1, let's see how it goes.

**Comment by RoRo-1** [comment]
Now is so from 30 min....
"Waiting for Solana approval..."

**Comment by RoRo-1** [comment]
I changed the amount to 0.064468. It's been over an hour and it still says "Waiting for Solana approval..."
This is the transaction:

3E21iRvMguNiXzGStqbYRhz6pqgKu3BbRAAaJp4eeFTAEHy7CmggUZZqEA8EMySP4w9tUUzUpcHniQqy5r8xqTuF

**Comment by RoRo-1** [comment]
What should I do?

**Comment by yamijuan** [comment]
Look at the recent transactions table. Connect both wallets and select the token to display the table. To a hard reload first shift + f5

**Comment by RoRo-1** [comment]
I checked the transaction table. 

3E21iRvMguNiXzGStqbYRhz6pqgKu3BbRAAaJp4eeFTAEHy7CmggUZZqEA8EMySP4w9tUUzUpcHniQqy5r8xqTuF

My transaction says on the Solana Network it ended successfully. 
I cleared the browser cache and connected the two wallets.
Now I don't see my WBTC tokens on my Metamask wallet in the Ethereum Network.
Where did the WBTC go?

**Comment by yamijuan** [comment]
do you see the tx in the table? wanna post an screenshot of the table

**Comment by leoluk** [comment]
@yamijuan reopened the issue in case there's anything you want to follow up on, feel free to close it otherwise

**Comment by RoRo-1** [comment]
I cleared the browser cache. I reloaded the V1 Bridge page. I connected the two wallets (Phantom and Metamask) to the bridge. On Phantom I selected WBTC (solana), while on Metamask the token WBTC is not present.
In practice the tokens are out of Solana but not entered in the Network Ethereum.
Now I'm trying to submit the signed VAA bytes with the submitVAA method. I have connected my Metamask but when I need to approve it tells me I have insufficient funds. But I have about $85 in my wallet and the total gas fee is $45. Why does it tell me I have insufficient funds?

**Comment by leoluk** [comment]
Screenshot of the error message?

**Comment by RoRo-1** [comment]
![insufficient-funds](https://user-images.githubusercontent.com/7474708/134357925-20dae553-8cf8-4700-889d-3abb13296483.jpg)

**Comment by leoluk** [comment]
According to Metamask docs, this indeed means that there's not enough gas to pay for the tx: https://metamask.zendesk.com/hc/en-us/articles/360044703372-How-to-fix-insufficient-funds-error-or-greyed-out-confirm-button

**Comment by RoRo-1** [comment]
Ok, it says insufficient funds but I have $85 in ETH in my wallet!

**Comment by leoluk** [comment]
Metamask in your screenshot says "0". Either way, we can't help with Metamask usage

**Comment by RoRo-1** [comment]
Of course, I understand this well, but I have about $85 in my portfolio. I don't know why it then tells me insufficient funds
![sufficient-funds](https://user-images.githubusercontent.com/7474708/134370731-2bb497c8-9c95-4625-9fce-a9784cdc6fb7.jpg)
.

**Comment by leoluk** [comment]
I think you need the 0.3 ETH listed as "max amount" but then again, I'm not a Metamask expert

**Comment by RoRo-1** [comment]
I have uploaded more ETH to the wallet. I have reached the amount of 0.048 ETH equal to $151. I entered the code in the submitVAA string and then "write".
This is the transaction ID:

0x38b60c4ab0a35d0640751ee8cab2aff23226bf88cfa23ef92f6716d95c9bd0bd

It looks like I did a 0 ETH move transaction but paying about $30 gas fee.
The important thing is that the WBT token arrived at its destination. 
Successful transaction.
Thank you
